std/math.d

```diff
@@ -5058,8 +5059,8 @@ real remquo(real x, real y, out int n) @trusted nothrow @nogc  /// ditto
         assert(remainder(-5.1, 3.0).feqrel(0.9) > 16);
         assert(remainder(0.0, 3.0) == 0.0);
 
-        assert(remainder(1.0, 0.0) is -real.nan);
-        assert(remainder(-1.0, 0.0) is -real.nan);
+        assert(isNaN(remainder(1.0, 0.0)));
+        assert(isNaN(remainder(-1.0, 0.0)));
```

Comment:
[Discussed in a previous comment on the PR when it was added](https://github.com/dlang/phobos/pull/6420#discussion_r190309079).